cmd/copy.go

```diff
@@ -38,6 +39,9 @@ var copyCmd = &cobra.Command{
 		}
 		remotepath := cmd.Flag("remotepath").Value.String()
 		destpath := cmd.Flag("destpath").Value.String()
+		if destpath != "/" {
```

Comment:
It is better to do it in gosdk or blobber

Reply:
done